### ISSUE TITLE
Invalid context call for pgpass

### ISSUE THREAD

**Issue by vitaly-t** [brianc/node-postgres#2319]
https://github.com/brianc/node-postgres/blob/acfbafac82641ef909d9d6235d46d38378c67864/packages/pg/lib/client.js#L220

this looks like invalid code, referencing `this` from inside a callback function that does not receive any context.

I've started getting issues logged against that...

```
Uncaught TypeError: Cannot read property 'connectionParameters' of undefined
      at /home/gitlab-runner/builds/tovhxyJG/0/muc/vdwh/npm-packages/db/node_modules/pg/lib/client.js:220:16
      at Object.onEnd (node_modules/pgpass/lib/helper.js:122:9)
```

**Comment by vitaly-t** [comment]
[This PR should fix it](https://github.com/brianc/node-postgres/pull/2320).

**Comment by brianc** [comment]
Fixed with `pg@8.3.3`!
